functions/user.js

```diff
@@ -1,80 +1,74 @@
-const config = require("./config");
+const _ = require("lodash");
 const GitHubbub = require('./githubbub');
-const util = require("util");
 const Promise = require('bluebird');
-const _ = require("lodash");
+const ServerValue = require('firebase-admin').database.ServerValue;
```

Comment:
Regarding the latter, turns out that `ServerValue` is actually only a property on the `admin.database` namespace, not on `admin.database.Database`.

See:
https://firebase.google.com/docs/reference/admin/node/admin.database
https://firebase.google.com/docs/reference/admin/node/admin.database.Database

Will make the other change you suggested.